src/parsers/js/transformers/eslint1/index.js

```diff
@@ -0,0 +1,23 @@
+import {defineRule, runRule} from '../../utils/eslintUtils';
```

Comment:
Oh, good catch! I thought Webpack would automatically make a new bundle for every `transformers/*/index.js` but maybe it doesn't work that way. How does the code splitting work? It is simply the `require([..], cb)` that does it? I've never used that feature of Webpack.